test/dynamo/test_misc.py

```diff
@@ -7631,6 +7631,58 @@ def my_dyn_fn(x):
 
         torch._dynamo.optimize("eager")(my_dyn_fn)(y)
 
+    @torch._dynamo.config.patch(specialize_float=False, capture_scalar_outputs=True)
+    def test_unspecialized_float_multiply_precision_bf16(self):
+        def fn(x, y):
+            return x * y
+
+        cnt = CompileCounterWithBackend("aot_eager")
+        fn_opt = torch._dynamo.optimize(cnt)(fn)
+        x = torch.tensor(9.734375, dtype=torch.bfloat16, requires_grad=True)
+        y = 1.00048828125
+
+        self.assertEqual(fn_opt(x, y), fn(x, y))
+        self.assertEqual(cnt.frame_count, 1)
```

Comment:
curious what are we trying to test here we only call fn_opt() , so its not recompilation that we are looking at?
are you tryin to assert that there is no graph breaks? if so maybe pass fullgraph= True?
or is it something related to forward and backward?